src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -302,7 +317,7 @@ private string GetAnalyzerPath(AnalyzerReference analyzerReference)
             return analyzerReference.FullPath;
         }
 
-        protected override void ApplyParseOptionsChanged(ProjectId projectId, ParseOptions options)
+        protected override void ApplyCompilationOptionsChanged(ProjectId projectId, CompilationOptions options)
```

Comment:
nit: can you put this method below ApplyParseOptionsChanged, that way hte diff is more readable?  Thanks!

Reply:
I'm sorry but no, the order is "compilation" first in a bunch of places already and I would prefer to keep it consistent. it's really not that bad, these methods are short.

Reply:
I would def like to get Jason's thoughts on this.  If this is ok, then i'm totally ok with it.  It just seemed like a more sensible approach to me.  Thanks!

Reply:
➡️ After further changes, the diff is readable now.